### PR TITLE
update tempalte setup as per flutter tutorial

### PR DESCRIPTION
**Pull request by MongoCaleb**
### Jira

https://jira.mongodb.org/browse/DOCSP-27595

### Staged Changes

https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/docsp-27595/tutorial/dotnet/

### CODE REVIEW

File: source/tutorial/dotnet.txt [hunk 1]
```diff
@@ -74,23 +74,62 @@ Prerequisites
...
+
+      From {+cli+}, you can run the :ref:`realm-cli apps create 
+      <realm-cli-apps-create>` command to set up the backend and create the 
+      C# (MAUI) base app. The following command creates a new app based on the 
+      ``maui.todo.flex`` template. With the options flags we can pass during 
+      creation, the command below creates an app:
+
+      - Named "MyTutorialApp"
+      - Deployed in the US-VA region
+      - With the environment set to "development" (instead of production or QA)
```
Comment (mongodben) [inline source/tutorial/dotnet.txt:115]: should you mention deployment model as well? there's a flag for it below.

alternatively, we could remove that `--deployment-model global` flag b/c i don't think it's necessary to get into global vs. local apps in the tutorial.

no strong opinion on approach here

File: source/tutorial/dotnet.txt [hunk 2]
```diff
@@ -74,23 +74,62 @@ Prerequisites
...
+      :guilabel:`C# (.NET MAUI)` client code.
+
+   .. tab:: App Services CLI
+      :tabid: cli
+
+      From {+cli+}, you can run the :ref:`realm-cli apps create 
+      <realm-cli-apps-create>` command to set up the backend and create the 
+      C# (MAUI) base app. The following command creates a new app based on the 
+      ``maui.todo.flex`` template. With the options flags we can pass during 
+      creation, the command below creates an app:
```
Comment (mongodben) [inline source/tutorial/dotnet.txt:111]: > With the options flags we can pass during creation, the command below creates an app:

this sentence reads odd to me. how about instead:

```rst
The below command creates an App with the following properties:
```

File: source/tutorial/dotnet.txt [hunk 3]
```diff
@@ -74,23 +74,62 @@ Prerequisites
...
+      :tabid: github
+
+      You can fork and clone a GitHub repository that contains the Device 
+      Sync client code. The C# client code is available at
+      :github:`https://github.com/mongodb/template-app-maui-todo <mongodb/template-app-maui-todo>`.
+
+      If you use this process to get the client code, you must create a
+      template app to use with the client. Follow the instructions at 
+      :ref:`create-template-app` to use the Atlas App Services UI, {+cli+},
+      or Admin API to create a Device Sync template app.
```
Comment (mongodben) [inline source/tutorial/dotnet.txt:132]: might want to say in here which app it is you're supposed to create. i believe its the `sync.todo` template app list on the template app reference page (https://www.mongodb.com/docs/atlas/app-services/reference/template-apps/)

**Review by mongodben — COMMENTED** [review]
looks pretty good! few small things throughout.

File: source/tutorial/dotnet.txt (hunk 2 above)
Comment (MongoCaleb) [reply source/tutorial/dotnet.txt:111]: I prefer "command below" over "below command". Also tweaked the wording of bullet 3 a bit. LMK your thoughts.

File: source/tutorial/dotnet.txt (hunk 3 above)
Comment (MongoCaleb) [reply source/tutorial/dotnet.txt:132]: In the UI it is called "Realm-time Sync app" so I changed the text here accordingly.

Comment (mongodben) [reply source/tutorial/dotnet.txt]: but this sentence is about UI, CLI, or API. does 'Real-time Sync template app' make sense in the CLI or API contexts?

**Review by mongodben — COMMENTED** [review]
looking good, just 1 remaining question

**Review by mongodben — APPROVED** [review]

**Pull request merged**